backend/application/advertisement.ts

```diff
@@ -53,6 +53,26 @@ export class AdvertisementApplicationService {
       )
     }
   }
+  async getUserAdvertisments() {
+    try {
+      const BannerAd = await prisma.advertisement.findMany({
```

Comment:
```suggestion
      const bannerAd = await prisma.advertisement.findMany({
```

冒頭は小文字にしましょう。
class名やType名、定数名は大文字でお願いします。

js tsのお作法なので覚えましょう。